pkg/cmd/roachtest/inconsistency.go

```diff
@@ -22,6 +22,7 @@ func registerInconsistency(r *testRegistry) {
 	r.Add(testSpec{
 		Name:       fmt.Sprintf("inconsistency"),
 		Owner:      OwnerKV,
+		Skip:       "Uses RocksDB put command; unskip when that's bypassed",
```

Comment:
TODO(adityamaru): Check if this has been unskipped since I picked the version of roachtest from master.

Reply:
For consistency, I am going to rely on engineers backporting all their changes since October 22nd themselves. I will ping storage since this caught my eye but I suspect there could be others.